src/StockportGovUK.NetStandard.Gateways.csproj

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="2.2.0" />
     <PackageReference Include="Polly" Version="7.2.0" />
-    <PackageReference Include="StockportGovUK.NetStandard.Models" Version="3.7.1" />
+    <PackageReference Include="StockportGovUK.NetStandard.Models" Version="3.7.1-prerelease-110" />
```

Comment:
If models is good and tested we need to push the latest version and consume a non pre release version in this application